tedge/src/system_services/managers/bsd.rs

```diff
@@ -181,5 +181,7 @@ fn as_service_name(service: SystemService) -> &'static str {
         SystemService::Mosquitto => "mosquitto",
         SystemService::TEdgeMapperAz => "tedge-mapper-az",
         SystemService::TEdgeMapperC8y => "tedge-mapper-c8y",
+        SystemService::TEdgeSMMapperC8Y => "tedge-mapper-sm-c8y",
+        SystemService::TEdgeSMAgent => "tedge-agent",
     }
 }
```

Comment:
This should be implemented once and for all in `SystemService`.

* This piece of code is duplicated for `systemd`, `bsd` , `openrc` ... and even `impl Display for SystemService`.
* It's really unlikely that we will use different names for one init service.